### PR TITLE
chore: Fix and update tooling

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: www/config.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+// Working around benmosher/eslint-plugin-import#1396.
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (bpas247) [inline www/config.js]: Hmm, I thought [this](https://github.com/react-bootstrap/react-bootstrap/pull/3977/commits/4957ab6ea9d1901604946dcbc1342064d110e200) commit fixed the `import/no-extraneous-dependencies` linting issue.

Comment (taion) [reply www/config.js]: Yes, but it's doing so at the cost of using deprecated modules on npm that we shouldn't be using, e.g. https://www.npmjs.com/package/crypto.

This is actually an upstream bug; we shouldn't be shadowing the built-ins.

Comment (bpas247) [reply www/config.js]: Maybe we can update this branch and fix the upstream bug in this PR?

Comment (taion) [reply www/config.js]: I mean, I looked at it a bit, but I couldn't figure out why this was happening

**Comment by taion** [comment]
okay, not actually a bug – we just needed to switch to the `node` import resolver. can someone take a look?

**Pull request merged**